pkg/autoscaler/autoscaler_test.go

```diff
@@ -463,12 +463,34 @@ func TestAutoscaler_UpdateTarget(t *testing.T) {
 	a.expectScale(t, now, 100, true)
 }
 
+func TestAutoScaler_Concurrency(t *testing.T) {
```

Comment:
Could you make this func name more self-descriptive and add some comments to explain why 1 pod is the desired number?

Reply:
Done.  (Also adjusted the numbers a little so that each number is unique to make it easier to read.)